### DISCUSSION TITLE
How can I style the <span> that wraps the individual skeleton element

### DISCUSSION THREAD

**Discussion by markcnunes** [dvtng/react-loading-skeleton#170] [Q&A] [answered]
I see the skeleton can be styled as `<Skeleton style={...} />`, `<Skeleton /* props such as width */ />` or const `StyledSkeleton = styled(Skeleton)` but all these 

```html
<span aria-live="polite" aria-busy="true"> <-- I need to add `width: 100%;    height: 100%;` to this span
  <span class="react-loading-skeleton">&zwnj;</span>
  <br>
</span>
```

All these approaches are only styling the `<span class="react-loading-skeleton">` but how can I target the `<span aria-live="polite" aria-busy="true">`? Is `containerClassName` meant for this? If yes, is there another approach? I am using styled-components and it feels a bit counterintuitive that I need to use a parent to target the `containerClassName` to be able to style this span.

**Comment by srmagura** [comment, marked as answer]
`containerClassName` is the intended way to style the `<span aria-live="polite" aria-busy="true">`. 

In Emotion, there is a [`ClassNames` component](https://emotion.sh/docs/class-names) for this exact use case. Can you check the styled-components documentation to see if they have something similar?

**Comment by markcnunes** [comment]
Thanks for the quick answer, @srmagura. Interesting, I didn't know Emotion supported it, unfortunately, it seems [styled-components doesn't support it yet](https://github.com/styled-components/styled-components/issues/3179#issuecomment-904177267). I am not too strong about it but just a suggestion, perhaps there could be a prop `containerStyle` that styles the element `<span aria-live="polite" aria-busy="true">` but I will be using `containerClassName` for now.
